post-scan-actions/aws-python-email-notification/README.md

```diff
@@ -0,0 +1,262 @@
+# Post Scan Action - Email Notification
+
+After a scan occurs, this example Lambda function sends out a notification Email using Amazon Simple Email Service when a malicious file got detected.
+
+## Prerequisites
+
+1. **Install supporting tools**
+    - Install the AWS command line interface (CLI). See [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for details.
+    - Install GNU Make if you don't want to use the AWS CLI. See [GNU Make](https://www.gnu.org/software/make/) for download information.
+2. **Configure Amazon Simple Email Servie**
+    - Verify the Email address you are going to use for sending the Email notifications
```

Comment:
Doesn't the recipient mail address need to be verified too?

Reply:
You're correct.

Reply:
Changed